src/eos-updater.c

```diff
@@ -219,6 +221,62 @@ listen_on_session_bus (void)
   return value != NULL;
 }
 
+/* Remove our configuration files from /etc which are identical to the current
+ * versions installed in /usr/share or /usr/etc, which we embed as an MD5
+ * checksum. If we do this on all systems, we can eventually change the formats
+ * in /usr/etc without worrying about the new defaults being overwritten by
+ * stale files in /etc.
+ *
+ * This functionality can be removed after a few releases, once we’re confident
+ * all systems will have been upgraded. */
```

Comment:
For what it's worth, if you remove the file from /etc in the new ostree and the local version is unchanged from the old ostree, then it will be removed. It's only when there are local modifications that it will hang around.

That said, this is a nice thing for the service to do.